docs/version0.15.md

```diff
@@ -65,10 +66,19 @@ The ChaCha20 and ChaCha20-Poly1305 algorithms can now use OpenSSL on Java 11. Fo
 
 For compatibility, the [`-XX:OnOutOfMemoryError`](xxonoutofmemoryerror.md) OpenJDK Hotspot option is now supported by OpenJ9.
 
-### Support for Transparent HugePage
+###  ![Start of content that applies only to Java 11 (LTS)](cr/java11.png) Support for Transparent HugePage
 
 The VM now supports the allocation of huge pages on Linux when you use the `madvise` (`/sys/kernel/mm/transparent_hugepage/enabled`) setting. To enable this feature, set [`-XX:+TransparentHugePage`](xxtransparenthugepage.md) on the command line when you start your application. This option is currently not enabled by default.
 
+### Support for low-overhead heap profiling
+
+[JEP 331](http://openjdk.java.net/jeps/331) provides a mechanism for sampling Java heap allocations with minimal resources, which can be achieved by using the JVM Tools Interface (JVMTI).
```

Comment:
Suggestion for changes: _provides a low-overhead way of sampling Java heap allocations via JVMTI._

Reply:
I'm finding "low-overhead way" a bit awkward. How about:

[JEP 331](http://openjdk.java.net/jeps/331) provides a mechanism for sampling Java heap allocations with a low overhead via the JVM Tools Interface (JVMTI).

Reply:
nitpick: _JVM Tools Interface (JVMTI)_ -> Java Virtual Machine Tool Interface (JVM TI)

Reply:
Done